docs/en_US/Tutorial/ExperimentConfig.rst

```diff
@@ -279,6 +280,15 @@ Optional. Integer between 1 and 99999. Default: 99999.
 
 Specifies the max number of trial jobs created by NNI, including succeeded and failed jobs.
 
+maxTrialDuration
+^^^^^^^^^^^^^^^^
+
+Optional. String. Default: 999d.
+
+**maxTrialDuration** specifies the max duration time of each trial job. The unit of the time is {**s**\ , **m**\ , **h**\ , **d**\ }, which means {*seconds*\ , *minutes*\ , *hours*\ , *days*\ }.
+
+Note: The maxTrialDuration spec set the time of each trial job. If current trial job reach the max duration time, this trial job will stop.
```

Comment:
you can remove "Note: ", because this is a normal description of "maxTrialDuration"

Reply:
Removed